srcpkgs/foliate/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'foliate'
+pkgname=foliate
+version=1.1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gjs-devel glib-devel"
```

Comment:
you porbably onlyneed gjs, not gjs-devel

Reply:
Indeed, thanks.